### PR TITLE
feat(perf-issues): document new threshold for consecutive db detector

### PR DESCRIPTION
**Pull request by DominikB2014**
A new threshold has been added to consec db detector recently, this PR documents the new threshold.

### CODE REVIEW

File: src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx [hunk 1]
```diff
@@ -22,6 +22,7 @@ The detector for this performance issue looks for a set of sequential, non-overl
 Once these spans are found, the following must also hold true:
 
 - Maximum time saved from parallelization must exceed a threshold
+- The ratio between the maximum time saved and the duration of the sequential spans must exceed a threshold
```
Comment (udameli) [inline src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx:25]: are these thresholds documented anywhere? might make sense to put them here for clarity

Comment (DominikB2014) [reply src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx:25]: The actual values themselves?

Comment (DominikB2014) [reply src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx:25]: If your referring to the values, they intentionally weren't added here for EA, as the thresholds could be tweaked. They will be added for GA

Comment (udameli) [reply src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx:25]: yeah the value themselves. Ah ok, gotcha 👍

**Review by udameli — APPROVED** [review]

**Pull request merged**